source/includes/store_me_groups/_header.md

```diff
@@ -0,0 +1 @@
+# Store Me Groups
```

Comment:
store me group files aren't shown as not needed for ISS

Reply:
that double negative is throwing me. Anything Store Me should be removed from docs from an M&E standpoint

Reply:
i just mean the file is there, but not displayed as it isn't listed in the index. i can either remove completely, or leave them in case needed in future in which case we'd just add it to the list in index.html.md